abnf/odata-aggregation-testcases.yaml

```diff
@@ -752,7 +752,7 @@ TestCases:
 
   - Name: hierarchy transformations - traverse
     Rule: queryOptions
-    Input: $apply=traverse(SalesOrgHierarchy,preorder)
+    Input: $apply=traverse($root/SalesOrganizations,SalesOrgHierarchy,ID,preorder)
```

Comment:
I would have expected to surround the first three with parens to make them a triple, just as in `rollup()`

Reply:
No, it's not like in `rollup` but like in `ancestors`.